CHANGELOG.md

```diff
@@ -5,8 +5,12 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+-   ##1576 : Add test functions which pass and return classes.
```

Comment:
You don't need to document the addition of tests in the CHANGELOG. Remember, the changelog is there for users to work out which version of the code they should be using, and for devs to find developments

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1576/commits/b6f1c7054ac8904ecf769c09923be7befd883a19.